src/constants/validateSchema.ts

```diff
@@ -0,0 +1,50 @@
+import * as yup from 'yup';
+
+export const schema = yup.object().shape({
+  name: yup
+    .string()
+    .required('Name is a required field')
+    .matches(/^[A-Z]/, 'Name must start with an uppercase letter'),
```

Comment:
This will not work with non-english letters. It would be better to use:

```typescript
test('Name requirement', 'Name must start with an uppercase letter', (value) => {
    return value.length > 0 && value[0].toUpperCase() == value[0] && value[0].toLowerCase() != value[0]
}
```